fix: stop displaying emails

ojeytonwilliams opened the pull request:
- fix: never show emails in the app
- test: update to use names not emails

<!-- Please follow the below checklist and put an `x` in each of the boxes to agree with the statement, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

<!-- If your pull request closes a GitHub issue, replace the XXXXX below with the issue number. For e.g. Closes #12. The issue #12 will automatically get closed when this PR gets merged. -->

To protect user privacy this stops the api from returning email addresses.  They're still used by the app, but there's no need for any user (up to and including the owner) from seeing them.

<!-- Tell us in detail about the changes you made and how it will affect the platform. -->

ojeytonwilliams committed:
fix: never show emails in the app

ojeytonwilliams committed:
test: update to use names not emails

Sboonny reviewed (approved):
LGTM 🎉, thank you for your contribution to the page! 👍.

ojeytonwilliams committed:
fix(test): use correct user type

ojeytonwilliams committed:
fix(test): stop requesting email field

ojeytonwilliams committed:
fix(test): get the chapter members from the db

Since the api no-longer returns emails, we have to bypass it and fetch
them from the db.

ojeytonwilliams committed:
fix(test): replace GQL queries with DB queries

gikf commented:
Reminders are printing emails to console, does this count? https://github.com/freeCodeCamp/chapter/blob/main/server/reminders/reminders.ts

ojeytonwilliams committed:
fix(test): banish getDashboardEventUsers

ojeytonwilliams committed:
fix: stop logging email addresses

Co-authored-by: Krzysztof G <60067306+gikf@users.noreply.github.com>

ojeytonwilliams commented:
Nice catch, @gikf. It's probably fine, but let's err on the side of caution.  After all, I'm not 100% convinced we'll need to log that info.
